src/App.css

```diff
@@ -13,8 +15,9 @@
   flex-direction: column;
   align-items: center;
   justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+  font-size: 1.6rem;
+  /* font-size: calc(10px + 2vmin); */
```

Comment:
Could probably get rid of commented out code.

Reply:
Done!